_emails/README.md

```diff
@@ -12,10 +12,9 @@ When adding templates, please ensure you are using UTF-8 encoding. Please do not
 | name            | yes      | String | Name of the email that will be displayed to users                                   |
 | city            | yes      | String | City the email template is for                                                      |
 | state           | yes      | String | State (abbreviated) the email template is for (e.g., NY)                            |
-| recipients      | yes      | Array  | Array of email addresses the email will be directly sent to                         |
+| recipients      | yes      | Array  | Array of email addresses the email will be directly sent to via bcc                 |
```

Comment:
another approach to this PR is to just update all the templates that use cc.. and move their cc to recipients. (but this can also be a follow on to this PR, in two parts)